mobile/ebitenmobileview/input_ios.go

```diff
@@ -58,3 +58,22 @@ func UpdateTouchesOnIOS(phase int, ptr int64, x, y int) {
 		panic(fmt.Sprintf("ebitenmobileview: invalid phase: %d", phase))
 	}
 }
+
+func UpdatePressesOnIOS(phase int, keyCode int, keyString string) {
+	switch phase {
+	case C.UITouchPhaseBegan, C.UITouchPhaseMoved, C.UITouchPhaseStationary:
+		if key, ok := iosKeyToUIKey[keyCode]; ok {
+			keys[key] = struct{}{}
+		}
+		runes = []rune(keyString)
```

Comment:
```go
if phase == C.UITouchPhaseBegan {
    for _, r := range keyString {
        if !unicode.IsPrint(r) {
            continue
        }
        runes = append(runes, r)
    }
}
```

Reply:
Why only in UITouchPhaseBegan?

Trying out if this is necessary, or even correct, by checking what phase is used for key repeat.

Reply:
Here in the Simulator, I get no events at all for key repeat. Let's actually do that instead.

Reply:
Hmm, I see. I think we should consider repeating (so the current input_android.go has the same issue), but is no event is depetected by repeating, there is nothing we can do. Let's handle only `UITouchPhaseBegan` to avoid unexpected behaviors.

Reply:
Done that now.

Also moved clearing of runes to after UpdateInput(). This is possibly an existing bug with Android - it might be possible that touch events can repeat the last key event. Don't know for sure though.

MIGHT be related to my not yet tracked down https://github.com/divVerent/aaaaxy/issues/222

Reply:
> Also moved clearing of runes to after UpdateInput(). This is possibly an existing bug with Android - it might be possible that touch events can repeat the last key event. Don't know for sure though.

Sorry but I realized this comment now. I'll take a look at Android later anyway.